src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -378,7 +429,8 @@ object SparkPlan {
       relations: Seq[Relation],
       rawPlan: String, // null for now
       sql: String, // null for now
-      fingerprint: LogicalPlanFingerprint)
+      fingerprint: LogicalPlanFingerprint,
+      lastFileId: Long = UNKNOWN_FILE_ID)
```

Comment:
As we discussed, could you replace with this properties Map[String,String] and keep the field with it? +"max" id

Reply:
Sure, added it.